README.md

```diff
@@ -1,25 +1,54 @@
-# Pixel
+# ACM@UIUC Pixel
 
 <img src="photo.jpg" width=500>
 
-A 128x128 crowdsourced display.
+A 128x128 crowd-sourced display.
+
+# Endpoints
+
+Base API URL: [http://pixel.acm.illinois.edu](http://pixel.acm.illinois.edu).
+
+Rate Limit: 2 requests per minute.
+
+## POST /image/pixel/
+
+Displays the image contained at the URL on the ACM@UIUC Pixel display.
 
 Send a POST request to `pixel.acm.illinois.edu` with data
+
 - `x`: 0-127
 - `y`: 0-127
-- `color`: #FFFFFF or a supported [color string](https://www.tcl.tk/man/tcl8.6/TkCmd/colors.htm)
+- `color`: #FFFFFF or a supported [color string](https://www.tcl.tk/man/tcl8.6/TkCmd/colors.htm).
 
-## Examples
+Request body:
+
+```
+{
+    "x": 25,
+    "y": 25,
+    "color": "#FF0000"
+}
+```
 
-curl
+## GET /image/screenshot/regular/
 
+Returns a regular-sized screenshot of the Pixel display.
+
+## GET /image/screenshot/small/
```

Comment:
How about just `/small.png` and `/regular.png`?

Reply:
No










(just kidding haha)
I can remove the image part, but I feel like screenshot has semantic value.